build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -194,6 +192,10 @@ fun Test.addOsAsInputs() {
     inputs.property("operatingSystem", "${OperatingSystem.current().name} ${System.getProperty("os.arch")}")
 }
 
+fun Test.isUnitTest() = listOf("test", "writePerformanceScenarioDefinitions", "writeTmpPerformanceScenarioDefinitions").contains(name)
```

Comment:
Is there a better way to do this, e.g. by looking at the source set? @jjohannes

Reply:
Since this was only moved and not added in this PR (see a few lines below) I would keep it like this. There might be no elegant way right now. We have a few places like this in this part of the build logic which could use some love - https://github.com/gradle/gradle-private/issues/3126.